### PR TITLE
My Portfolio page 1

### PR DESCRIPTION
**Pull request by zakiyousufi**
I creat the first page of my portfolio 
I follow the requirement and used the Figma template
there is no linters error 
My code is clean 
I follow the GitHub flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+Prerequisites
+A browser
+
+Usage
+Clone the repositry and open the index.html which is in the folder.
+
+## Authors
+GitHub: @zakiyousufi
+Twitter: @ahmadzakiyousofi
+LinkedIn: ahmadzakiyousufi
```
Comment (zieeco) [inline README.md]: **[REQUIRED  CHANGES]**

Please ensure your social media account is a link that takes a user to the referred account.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+</head>
+
+<body>
+    <header class="header">
+        <nav class="statusbar">
+            <img class="logo" alt="My logo picture" src="css/image/logo/5309769.png">
+            <img class="menu" alt="Menu bar" src="css/image/Bars/Menu.svg">
+        </nav>
+    </header>
+    <div class="page1-content blackpage">
```
Comment (zieeco) [inline index.html]: **[REQUIRED  CHANGES]**

- To enforce the use of HTML5 semantic tag, please kindly use a `<section>` tag here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    <img class="due icons" alt="Social Media Icon" src="css/Icons/due.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="tweeter icons" alt="Tweeter Icon" src="css/Icons/tweeter.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="m-icon icons" alt="Social Media Icon" src="css/Icons/M.svg">
+                </a>
+            </div>
+            <div>
```
Comment (zieeco) [inline index.html]: **[REQUIRED  CHANGES]**

- To enforce the use of HTML5 semantic tag, please kindly use a `<ul>` tag to wrap your list items like this:

```
<ul>
  <li>
    <a href="">
      <img src="" alt="">
    </a>
  </li>
</ul>

```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @zakiyousufi 


Great work on your portfolio setup 👏 
You've done well implementing the design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @zakiyousufi, 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zakiyousufi** [comment]
@Ginohmk 
Thank you very much for reviewing my project 
cheers

**Pull request merged**